OpenStack-nova-accounting/extract_accounting.py

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/python
+import time
+import datetime
+#from datetime import datetime,time
+import json
+import requests
+import sys
+import sqlalchemy
+from sqlalchemy.sql import select
+from sqlalchemy.orm import sessionmaker
+from ConfigParser import SafeConfigParser
+
+
+# Read from config file
+parser = SafeConfigParser()
+try:
+   parser.read('/etc/influxdb.conf')
+   host = parser.get('db', 'host')
+   database = parser.get('db', 'database')
+   username = parser.get('auth', 'username')
+   password = parser.get('auth', 'password')
+   instance = parser.get('cloud','instance')
+except:
+   print 'Unable to read from config file'
+   sys.exit(1)
+
+url = 'http://'+host+'/write?db='+database +'&precision=s'
+
+nowtime = time.localtime()
+
+try:
+   parser.read('/etc/nova/nova.conf')
+   connectionstring = parser.get('database','connection')
+except:
+   print 'Unable to read from config file'
```

Comment:
Use
`raise RuntimeError("Unable to read from config file")`
That way it will log to syserr and exit at the same time

Reply:
Done